src/include/libpmem2.h

```diff
@@ -114,7 +114,7 @@ int pmem2_config_set_address(struct pmem2_config *cfg, unsigned type,
 	void *addr);
 
 enum pmem2_granularity {
-	PMEM2_GRANULARITY_BYTE,
+	PMEM2_GRANULARITY_BYTE = 1,
```

Comment:
This will make it difficult to use this enum as an array index/loop iterator. I'd prefer to add PMEM2_GRANULARITY_UNDEFINED or something like that.